### PR TITLE
Update Delegated Permissions Instructions

### PR DESCRIPTION
**Pull request by javadrum**
Included multiple selection note for all items in delegated permissions.  Added a note for items that require Admin Approval.

### CODE REVIEW

**Comment by PRMerger10** [comment]
@javadrum : Thanks for your contribution! The author, @markingmyname, has been notified to review your proposed change.

File: powerbi-docs/developer/embed-sample-for-customers.md [hunk 1]
```diff
@@ -83,9 +83,12 @@ You will need to enable additional permissions for your application in addition
 
     ![Select PBI Services](media/embed-sample-for-customers/embed-sample-for-customers-014.png)
 
-7. Select all permissions under **Delegated Permissions**. You will need to select them one by one in order to save the selections. Select **Save** when done.
+7. Select all permissions under **Delegated Permissions**. You can quickly select all **Delegated Permissions** by clicking the square next to the **Delegated Permissions** column header, or you can select them one by one. Select **Save** when done.
    
     ![Select delegated permissions](media/embed-sample-for-customers/embed-sample-for-customers-015.png)
+    
+    >[!Note]
+    >Granting some permissions will require an Admin approval.  These are denoted by a column labeled **Requires Admin**.  Items labeled yes with a green circle and check box.
```
Comment (markingmyname) [inline powerbi-docs/developer/embed-sample-for-customers.md]: @javadrum Thank you for your contribution.  Can you make some minor changes, please?

 Granting some permissions requires admin approval.  These are denoted by a column labeled **Requires Admin** with a green circle marked as *Yes*.

**Comment by PRMerger10** [comment]
@javadrum : Thanks for your contribution! The author, @markingmyname, has been notified to review your proposed change.

**Comment by PRMerger19** [comment]
@javadrum : Thanks for your contribution! The author, @markingmyname, has been notified to review your proposed change.

**Comment by PRMerger14** [comment]
@javadrum : Thanks for your contribution! The author, @markingmyname, has been notified to review your proposed change.

**Comment by javadrum** [comment]
updated

**Comment by markingmyname** [comment]
#sign-off

**Pull request closed without merging**